agenda/presentation/src/main/java/com/nibalk/tasky/agenda/presentation/components/AgendaCard.kt

```diff
@@ -65,11 +61,13 @@ fun AgendaCard(
         }
     }
 
-    val contentColor = if (item is AgendaItem.Task) TaskyWhite else TaskyDarkGray
-    val backgroundColor: Color = when (item) {
-        is AgendaItem.Task -> TaskyGreen
-        is AgendaItem.Event -> TaskyLightGreen
-        is AgendaItem.Reminder -> TaskyBrownLight
+    val cardColor: Pair<Color, Color> = when (item) {
```

Comment:
If you really want to work with pairs here, I would at least make use of destructuring. Pairs sadly make the code very unreadable, since they don't communicate by design what each value stands for. With destructuring you can achieve that:
```
val (cardBackgroundColor, cardContentColor) = when(item) { ... }
```